examples/03_connectivity/plot_seed_based_connectome.py

```diff
@@ -1,57 +1,89 @@
+# We fetch the coordinates of Power atlas
+from nilearn import datasets
+
+power = datasets.fetch_coords_power_2011()
+print(power.keys())
+
+###############################################################################
+# and the coordinates, regions labels and networks names of the Dosenbach
+# atlas.
+dosenbach = datasets.fetch_coords_dosenbach_2010(ordered_regions=True)
+print(dosenbach.keys())
+
+###############################################################################
+# Note the use of the parameter *ordered_regions*. It allows to have ROIs
+# **ordered** with respect to **functional networks**, which come at number
+# of 6.
 import numpy as np
-import matplotlib.pyplot as plt
-from nilearn import datasets, connectome, plotting, input_data
 
+print(np.unique(dosenbach.networks))
 
 ###############################################################################
-# Atlas and dataset fetching
+# Computing within spheres averaged time-series
+# ---------------------------------------------
 
-# Fetch the coordinates of power atlas
-power = datasets.fetch_coords_power_2011()
+###############################################################################
+# We are going to use a single subject from the ADHD dataset.
+adhd = datasets.fetch_adhd(n_subjects=1)
+
+
+###############################################################################
+# We can compute the mean signal within **spheres** of a fixed radius around
+# a sequence of 3D coordinates with
+# :class:`nilearn.input_data.NiftiSpheresMasker`. 
+# So we collect the regions coordinates in numpy arrays
 power_coords = np.vstack((
     power.rois['x'],
     power.rois['y'],
     power.rois['z'],
 )).T
 
-dosenbach = datasets.fetch_coords_dosenbach_2010()
 dosenbach_coords = np.vstack((
     dosenbach.rois['x'],
     dosenbach.rois['y'],
     dosenbach.rois['z'],
 )).T
 
-# Fetch the first subject of ADHD dataset
-adhd = datasets.fetch_adhd(n_subjects=1)
-
-
 ###############################################################################
-# Masking: taking the signal in a sphere of radius 5 around Power coords
+# and define the spheres maskers, with small enough radius to avoid regions
+# overlap.
+from nilearn import  input_data
 
 power_masker = input_data.NiftiSpheresMasker(
     seeds=power_coords, smoothing_fwhm=4, radius=5.,
-    standardize=True, detrend=True,
-    low_pass=0.1, high_pass=0.01, t_r=2.5)
+    detrend=True, low_pass=0.1, high_pass=0.01, t_r=2.5)
 
 dosenbach_masker = input_data.NiftiSpheresMasker(
-    seeds=dosenbach_coords, smoothing_fwhm=4, radius=4.,
-    standardize=True, detrend=True,
-    low_pass=0.1, high_pass=0.01, t_r=2.5)
+    seeds=dosenbach_coords, smoothing_fwhm=4, radius=4.5,
+    detrend=True, low_pass=0.1, high_pass=0.01, t_r=2.5)
+
+###############################################################################
+# Voxel-wise time-series within each sphere are **averaged** and then
+# **preprocessed** in an All-In-One fashion.
```

Comment:
The word "preprocessed" is quite heavily connoted in fMRI. It's means things like spatial realignment. I would rather avoid using it. I would write something like:

"Voxel-wise time-series within each sphere are **averaged**. The resulting signal is then prepared by the masker object: detrended and band-pass filtered."